source/includes/examples/_cart-pole.html.md

```diff
@@ -2,7 +2,7 @@
 
 > ![Cart Pole Balance](../images/cart-pole-balance.gif)
 
-[**Download the full source code on GitHub**][1] so you can run it yourself on the Bonsai Platform.
+[**Download the full source code on GitHub**][1] if you want to run this simulator locally or create a new BRAIN on [beta.bons.ai][4] to run it yourself on the Bonsai Platform.
```

Comment:
This sentence is a little confusing to me, since 1) you need to create a BRAIN in both cases (local or remote), and 2) 'run it yourself' is a little ambiguous.

What something like
`[**Download the full source code on GitHub**][1] if you want to run this simulator locally. If you want to run Cart Pole remotely on the Bonsai Platform as a managed simulator, create a new BRAIN selecting the Cart Pole demo on [beta.bons.ai][4]`

It's worth noting that we call it `Cart Pole` here in the documentation, but the new BRAIN page calls it `Cartpole`. Should we open a task to fix it on the new BRAIN page? (Seems like it is generally treated as two separate words)